src/components/manage/Sidebar/ObjectBrowserBody.jsx

```diff
@@ -79,22 +83,37 @@ class ObjectBrowserBody extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      currentFolder: this.props.data.url
-        ? getParentURL(this.props.data.url)
-        : '/',
-      currentImageFolder: this.props.data.url
-        ? getParentURL(this.props.data.url)
-        : '/',
-      currentLinkFolder: this.props.data.href
-        ? getParentURL(this.props.data.href)
-        : '/',
+      currentFolder:
+        this.props.mode === 'multiple'
+          ? '/'
+          : this.props.data?.url
+          ? getParentURL(this.props.data.url)
+          : '/',
+      currentImageFolder:
+        this.props.mode === 'multiple'
+          ? '/'
+          : this.props.data?.url
+          ? getParentURL(this.props.data.url)
+          : '/',
+      currentLinkFolder:
+        this.props.mode === 'multiple'
+          ? '/'
+          : this.props.data?.href
+          ? getParentURL(this.props.data.href)
+          : '/',
       parentFolder: '',
-      selectedImage: this.props.data.url
-        ? this.props.data.url.replace(settings.apiPath, '')
-        : '',
-      selectedHref: this.props.data.href
-        ? this.props.data.href.replace(settings.apiPath, '')
-        : '',
+      selectedImage:
+        this.props.mode === 'multiple'
+          ? ''
+          : this.props.data?.url
+          ? this.props.data.url.replace(settings.apiPath, '')
```

Comment:
`flattenToAppURL` everywhere, crap some of them are mine 😱 Probably coming from before it existed 😅